FEXCore/Source/Interface/Core/Frontend.cpp

```diff
@@ -967,8 +967,14 @@ void Decoder::BranchTargetInMultiblockRange() {
     TargetRIP &= 0xFFFFFFFFU;
   }
 
-  // If the target RIP is within the symbol ranges then we are golden
-  if (TargetRIP >= SymbolMinAddress && TargetRIP < SymbolMaxAddress) {
+  // If the target RIP is x86 code within the symbol ranges then we are golden
+  bool ValidMultiblockMember = TargetRIP >= SymbolMinAddress && TargetRIP < SymbolMaxAddress;
+
+#ifdef _M_ARM_64EC
+  ValidMultiblockMember = ValidMultiblockMember && !RtlIsEcCode(TargetRIP);
```

Comment:
Nit: `ValidMultiblockMember &= !RtlIsEcCode(TargetRIP);`

Reply:
ah sure, suppose the ec check is cheap enough the lack of short circuiting doesn't really matter

Reply:
Oh.. right.. short circuiting might help perf here... mm, I guess this is ok as is then.